File: Core Java/collage-app/src/com/techlabs/collage/Student.java
package com.techlabs.collage;

import com.techlabs.enums.Branch;

public class Student extends Person {

	private Branch branch;

	public Student(int id, String address, String dob, Branch branch) {
		super(id, address, dob);
		this.branch = branch;
	}

	public Branch getBranch() {
		return branch;
	}

	@Override
	public String toString() {
		String details = "Student Details.......\n";
		details += "Id : "+getId()+ "\n";
		details += "Address : " +getAddress()+ "\n";
		details += "DOB : " +getDob()+ "\n";
		details += "Branch : " +getBranch()+ "\n";
		return details ;
	}
}

File: DOTNET/C#/CollageApp/CollageApp/Student.cs
using System;

namespace CollageApp
{
    public enum Branch
    {
        COMPUTER, INFORMATIONTECHNOLOGY, CIVIL, ELECTRIC,
        MECHANICAL, AUTOMOBIL, CHEMICAL, ARONOTICAL

    }
    public class Student : Person
    {
        private Branch branch;
        public Student(int id, string address, string dob, Branch branch) : base(id, address, dob)
        {
            this.branch = branch;
        }

        public Branch Branch { get { return branch; } }

        public override string ToString()
        {
            String details = "Student Details....\n";
            details += "Id: " + Id + "\n";
            details += "Address: " + Address + "\n";
            details += "DOB: " + Dob + "\n";
            details += "Branch: " + Branch + "\n";
            return details;
        }
    }
}
